website/docs/r/policy_nat_rule.html.markdown

```diff
@@ -80,6 +82,7 @@ The following arguments are supported:
 * `translated_networks` - (Optional) A list of translated network IP addresses or CIDR.
 * `translated_ports` - (Optional) Port number or port range. For use with `DNAT` action only.
 * `scope` - (Optional) A list of paths to interfaces and/or labels where the NAT Rule is enforced.
+* `policy_based_vpn_mode` - (Optional) Policy based VPN mode. One of `BYPASS`, `MATCH`
```

Comment:
Could we please specify here that this attribute is only relevant for certain types of NAT?

Reply:
Yes, I've added that it only applies to `DNAT` and `NO_DNAT`. Also I've added that the default for these actions is `BYPASS` and that this flag is supported from NSX 4.0 and above.